ios/Sources/AboutFeature/AboutDroidKaigiScreen.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+import Styleguide
+
+public struct AboutDroidKaigiScreen: View {
+    @Environment(\.presentationMode) var presentationMode
+
+    public init() {
+        UITableView.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().barTintColor = .clear
+    }
+
+    public var body: some View {
+        NavigationView {
+            VStack {
+                VStack {
+                    Text(L10n.AboutDroidKaigiScreen.whatIs).font(.system(size: 22))
+                    AssetImage.logoTitle.image
+                        .frame(width: 261.81, height: 41.06, alignment: .center)
```

Comment:
`alignment: .center` is default value. could be removed 🙏

Reply:
DONE